.goreleaser.yml

```
@@ -0,0 +1,24 @@
+project_name: connectconformance
+builds:
+  - id: connectconformance
+    binary: connectconformance
+    main: ./cmd/connectconformance
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    ldflags:
+      - -s -w -X main.buildVersion=v{{.Version}}
```

This won't work until we also merge #702. Also, this will need to be tweaked since the `buildVersion` var is not in the `main` package:
```suggestion
      - -s -w -X connectrpc.com/conformance/internal.buildVersion=v{{.Version}}
```